orchagent/main.cpp

```diff
@@ -13,42 +13,22 @@ extern "C" {
 #include <unistd.h>
 
 #include <sairedis.h>
+#include <logger.h>
+
 #include "orchdaemon.h"
-#include "logger.h"
+#include "saiapi.h"
 
 using namespace std;
 using namespace swss;
 
 extern sai_switch_notification_t switch_notifications;
 
-#define UNREFERENCED_PARAMETER(P)       (P)
+extern sai_switch_api_t *sai_switch_api;
+extern sai_router_interface_api_t *sai_router_intfs_api;
```

Comment:
Could you elaborate 'easy track'? Actually prefer collect all the extern sai_xxx_api into one header file, which is more clean code.